components/Section7/Section7.jsx

```diff
@@ -13,7 +13,7 @@ function Section7() {
         </div>
         <div className=" w-full h-[90%]  flex flex-col justify-center items-center">
           {/* motion animation */}
-          <div className="w-full h-[342.48px] -sm:h-auto relative flex mx-auto -sm:px-2 -md:px-4  mb-24 justify-center gap-4 md-bw:gap-16">
+          <div className="w-full h-[342.48px] -sm:h-auto relative flex mx-auto -sm:px-2 -md:px-4 -sm:mb-8  mb-20 justify-center gap-4 md-bw:gap-16">
```

Comment:
Never out floating point pixels.
It must be multiply of 2.
If the design is odd, always ask back to the designer.

Reply:
II appreciate your assistance so much ! I'll fix it rn